sashimi-webapp/src/database/sql-related/sqlCommands.js

```diff
@@ -214,384 +216,304 @@ function getListOfFilesIdsWithSamePath(folderPath) {
+      .then(data => resolve(getArray(data)))
+      .catch(sqlError => reject(sqlError))
+    );
   };
 
   this.loadFile = function loadFile(fileId) {
-    if (typeof Promise === 'function') {
-      return new Promise((resolve, reject) =>
-        alasql.promise([stringManipulator.stringConcat('SELECT ', constants.HEADER_FILE_MANAGER_FILE_MARKDOWN,
-                                                       ' FROM ', constants.ENTITIES_FILE_MANAGER,
-                                                       ' WHERE ', constants.HEADER_FILE_MANAGER_FILE_ID,
-                                                       ' = ', fileId)])
-        .then(data => resolve(getDataOutOfAlasql(data)))
-        .catch(sqlError => reject(sqlError))
-      );
-    } else {
-      throw new exceptions.PromiseFunctionNotDefined();
-    }
+    return new Promise((resolve, reject) =>
+      alasql.promise([stringManipulator.stringConcat('SELECT ', constants.HEADER_FILE_MANAGER_FILE_MARKDOWN,
+                                                      ' FROM ', constants.ENTITIES_FILE_MANAGER,
+                                                      ' WHERE ', constants.HEADER_FILE_MANAGER_FILE_ID,
+                                                      ' = ', fileId)])
+      .then(data => resolve(getDataOutOfAlasql(data)))
+      .catch(sqlError => reject(sqlError))
+    );
   };
 
   this.changeFileName = function changeFileName(fileId, newFileName) {
-    if (typeof Promise === 'function') {
-      return new Promise((resolve, reject) =>
-        alasql.promise([stringManipulator.stringConcat('UPDATE ', constants.ENTITIES_FILE_MANAGER,
-                                                       ' SET ', constants.HEADER_FILE_MANAGER_FILE_NAME,
-                                                       ' = "', newFileName,
-                                                       '" WHERE ', constants.HEADER_FILE_MANAGER_FILE_ID,
-                                                       ' = ', fileId)])
-        .then(() => resolve())
-        .catch(sqlError => reject(sqlError))
-      );
-    } else {
-      throw new exceptions.PromiseFunctionNotDefined();
-    }
-  };
-
-  this.changeFilePath = function changeFilePath(fileId, newPath) {
-    if (typeof Promise === 'function') {
-      return new Promise((resolve, reject) =>
+    return new Promise((resolve, reject) =>
       alasql.promise([stringManipulator.stringConcat('UPDATE ', constants.ENTITIES_FILE_MANAGER,
-                                                    ' SET ', constants.HEADER_FILE_MANAGER_PATH,
-                                                    ' = "', newPath,
-                                                    '" WHERE ', constants.HEADER_FILE_MANAGER_FILE_ID,
-                                                    ' = ', fileId)])
+                                                      ' SET ', constants.HEADER_FILE_MANAGER_FILE_NAME,
+                                                      ' = "', newFileName,
```

Comment:
I think someone either:
UI / FileManager/  Storage should handle illegal character here.

Storage should definitely clean/sanitize the input before inserting data here. 

#275